src/main/java/dataLoader/csvDataLoader.java

```diff
@@ -0,0 +1,33 @@
+package dataLoader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+/**
+ * Simple class that parses data set from a csv file
+ * without knowing the headers of the file
+ * */
+
+public class csvDataLoader {
+
+    private String location = "/path/to/file.csv";
+    private Reader csvData = new StringReader(location);
+
+    public void parseCsvData( String csv) throws IOException {
```

Comment:
we expect from a CSV data loader to parse CSV files....so you don't have to be explicit here. Simply write ```parseFile``` the same is true for the ```JsonDataLoader```. This way we have a established a contract for data loaders and perhaps in the future we can abstract some stuff

Reply:
you' re right. will change it in both classes.